examples/lib/stories/effects/opacity_effect_example.dart

```diff
@@ -43,10 +46,10 @@ class OpacityEffectExample extends FlameGame with TapDetector {
   @override
   void onTap() {
     final opacity = sprite.paint.color.opacity;
-    if (opacity == 1) {
-      sprite.add(OpacityEffect.fadeOut());
-    } else if (opacity == 0) {
-      sprite.add(OpacityEffect.fadeIn());
+    if (opacity >= 0.5) {
+      sprite.add(OpacityEffect.fadeOut(StandardEffectController(duration: 1)));
```

Comment:
This line is a little bit long now, I wonder if we couldn't create shortcuts or something (not for this PR ofc)